ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -225,7 +225,7 @@ class DrawerPresentationController: UIPresentationController {
             }
         case .up:
             if actualPresentationOrigin == containerView.bounds.maxY {
-                return containerView.safeAreaInsets.bottom + keyboardHeight
+                return keyboardHeight == 0 ? containerView.safeAreaInsets.bottom : keyboardHeight
```

Comment:
Can you verify this using the "Some editable content" keyboard test, but simulating a hardware keyboard?

Reply:
Like this? 

https://github.com/microsoft/fluentui-apple/assets/22566866/013b90a6-d942-4317-8e44-8924e6d4bd9f